### DISCUSSION TITLE
Weird behavior in carousel when switching from a map with a lot of difficulties

### DISCUSSION THREAD

**Discussion by Szago** [ppy/osu#16543] [Q&A] [answered]
**Describe the bug:** 
Frequently, when switching from a beatmap with 13+ difficulties, all of the difficulties briefly overlay themselves on top of other beatmaps in carousel before disappearing.

**Screenshots or videos showing encountered issue:**

https://user-images.githubusercontent.com/43099280/150467929-d610f6de-efcb-4805-a536-416eede5e589.mp4
Specific frame:
![add](https://user-images.githubusercontent.com/43099280/150468113-4e9ac44d-893b-4a80-93fe-ac31ed6d301c.png)

**osu!lazer version:** 2022.118.0

**Logs:**

[performance.log](https://github.com/ppy/osu/files/7910526/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7910527/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7910528/updater.log)
[network.log](https://github.com/ppy/osu/files/7910529/network.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/11065
